ios/GitPoint/AppDelegate.m

```diff
@@ -20,9 +20,9 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 {
   NSURL *jsCodeLocation;
 
-
+ 
```

Comment:
> the pure empty line

@machour Sorry, I didn't comment very clearly before.

Reply:
Originally there was a line with a single space in it. I just removed it all together.
Not sure if I'm following you though :D